RussianFilter/sections/specific.txt

```diff
@@ -31,6 +31,7 @@ azpc.info###captainjack
 doefiratv.net##div[style="display: table;width: 100%;height: 360px;background-color: #f1f1f1;"]
 brestcity.com##.BlockTop1
 brestcity.com###page > div#wrapper~a[rel="nofollow"] > img
+softsalad.ru##.hidden-xs > a[target="_blank"][rel="nofollow"]
```

Comment:
better make it more specific. Not all `a[target="_blank"][rel="nofollow"]` - ads.
`.adsbygoogle~div[class="hidden-xs"] > a[rel="nofollow"] > img`

Reply:
@Alex-302, done